service_configuration_lib/spark_config.py

```diff
@@ -93,19 +106,27 @@ def get_mesos_spark_env(
         spark_env['spark.mesos.uris'] = 'file:///root/.dockercfg'
 
     spark_env = {**spark_env, **user_spark_opts}
-    spark_env['spark.mesos.executor.docker.parameters'] = 'cpus={}'.format(spark_env['spark.executor.cores'])
+    spark_env['spark.mesos.executor.docker.parameters'] = 'cpus={},{}'.format(
+        spark_env['spark.executor.cores'],
+        spark_env['spark.mesos.executor.docker.parameters'],
+    )
     _validate_spark_env(spark_env, event_log_dir)
 
     return spark_env
 
 
+def _generate_volume_name(volume):
+    return volume['containerPath']
+
+
 def get_k8s_spark_env(
     spark_app_name: str,
     spark_ui_port: str,
     paasta_cluster: str,
     paasta_service: str,
     paasta_instance: str,
     docker_img: str,
+    volumes: List[Any],
```

Comment:
nit: can we make the volumes a List of `TypedDict`s?

Reply:
`TypedDict` cannot be used to declare argument type. I will use `Mapping` instead.